internal/common.go

```diff
@@ -20,8 +20,8 @@ const (
 	// ClientType is used in the Management
 	ClientType         = "GOO"
 	AggregateFactoryID = -29
-)
 
-// ClientVersion is the build time version
-// TODO: This should be replace with a build time version variable, BuildInfo etc.
-var ClientVersion = "1.0.0"
+	// ClientVersion is the build time version
+	// TODO: This should be replaced with a build time version variable, BuildInfo etc.
```

Comment:
Can we remove TODO's from the codebase and open issues for them if needed instead ?

Reply:
OK, will do that in a separate PR.